script.py

```diff
@@ -27,68 +27,67 @@ def generate_token(client_id, client_secret):
         return token_response.json()['access_token']
 
 
-def request_vehicles(id):
+def request_vehicles(bus_id=None):
     authorization_header = {'Authorization': f'Bearer {token}'}
-    beaverbus_vehicles = 'v1/beaverbus/vehicles/'
-    if id:
-        return requests.get(f'{osu_API}{beaverbus_vehicles}{id}',
+    beaverbus_vehicles = f'{osu_api}v1/beaverbus/vehicles/'
+    if bus_id:
+        return requests.get(f'{beaverbus_vehicles}{bus_id}',
                             headers=authorization_header
                             )
     else:
-        return requests.get(f'{osu_API}{beaverbus_vehicles}',
+        return requests.get(f'{beaverbus_vehicles}',
                             headers=authorization_header
                             )
```

Comment:
I think you missed this comment: https://github.com/kalcicd/Python-Beaver-Bus-API-request-script/pull/3#discussion_r305154990